packages/common/src/intl/locales/en/distribution.json

```diff
@@ -45,6 +46,7 @@
   "label.in-store": "In store",
   "label.edited-by": "Edited by",
   "label.issue": "Issue",
+  "label.no-unallocated-rows-selected": "All of the selected rows are already allocated",
```

Comment:
same approach as delete - you can select rows and try to act on them and are then told why it was a dumb idea.. we don't disable buttons  or otherwise prevent actions from happening.